src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,34 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import usersFromServer from '../../api/users';
+
+type TodoListProps = {
+  todos: {
+    id: number;
+    title: string;
+    completed: boolean;
+    userId: number;
+  }[];
```

Comment:
```suggestion
  todos: Todo[];
```
just import the Todo type and reuse it here